mods/ra/chrome/assetbrowser.yaml

```diff
@@ -1,5 +1,6 @@
 Background@ASSETBROWSER_PANEL:
 	Logic: AssetBrowserLogic
+		SupportedFormats: .shp, .r8, tmp, .tem, .des, .sno, .int, .jun, .vqa
```

Comment:
What does `tmp` mean here? Terrain tiles are in TMP (RA) format and not in SHP format, but still having the terrain-specific extension.